<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Content-based Music Recommender System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>-20</y>
      <width>861</width>
      <height>161</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="RecordRadio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>82</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Record</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="UploadRadio">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>82</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RecordButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>35</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normalon>record.png</normalon>
      </iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="UploadButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>104</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="UploadPath">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>104</y>
       <width>660</width>
       <height>40</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>30</y>
       <width>120</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLCDNumber" name="SelectTimeDisplay">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="numDigits">
         <number>2</number>
        </property>
        <property name="intValue" stdset="0">
         <number>30</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>seconds</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>46</y>
       <width>560</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>490</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Music Tag</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TagShow">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>485</y>
      <width>391</width>
      <height>30</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="FigureShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>861</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>453</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>142</y>
      <width>110</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>520</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recommended Music</string>
    </property>
   </widget>
   <widget class="QTableView" name="RecommendTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>861</width>
      <height>190</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>RecordRadio</sender>
   <signal>clicked()</signal>
   <receiver>UploadPath</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
